js/widgets/page.js

```diff
@@ -73,36 +73,15 @@ $.widget( "mobile.page", {
 		if ( this.options.enhanceWithin ) {
 			this.element.enhanceWithin();
 		}
-		// Dialog widget is deprecated in 1.4 remove this in 1.5
-		if ( $.mobile.getAttribute( this.element[ 0 ], "role" ) === "dialog" && $.mobile.dialog ) {
-			this.element.dialog();
-		}
 	},
 
 	_enhance: function() {
-		var attrPrefix = "data-" + $.mobile.ns,
-			self = this;
-
 		if ( this.options.role ) {
```

Comment:
Based on our recent discussion around the enhanced option this should run either way while its dom manip its not creation of DOM and it does not effect layout or rendering.